### PR TITLE
TablePanel: display multi-line text

### PR DESCRIPTION
**Pull request by michael-az**
**What this PR does / why we need it**:
This pull request makes it possible to use multiline text in the TablePanel. Unnecessary line breaks were removed.
(when `Preserve Formatting` in Column Styles is set to on)

**Which issue(s) this PR fixes**:
similar to #17584

### CODE REVIEW

File: public/app/plugins/panel/table/renderer.ts [hunk 1]
```diff
@@ -280,26 +280,23 @@ export class TableRenderer {
...
       cellClasses.push('table-panel-cell-link');
 
-      columnHtml += `
-        <a href="${cellLink}" target="${cellTarget}" data-link-tooltip data-original-title="${cellLinkTooltip}" data-placement="right"${textStyle}>
-          ${value}
-        </a>
-      `;
+      columnHtml += `<a href="${cellLink}" target="${cellTarget}" data-link-tooltip data-original-title="${cellLinkTooltip}" data-placement="right"${textStyle}>`;
+      columnHtml += `${value}`;
+      columnHtml += `</a>`;
```
Comment (torkelo) [inline public/app/plugins/panel/table/renderer.ts]: Can you describe you change here? why remove the indenting?

File: public/app/plugins/panel/table/renderer.ts [hunk 2]
```diff
@@ -280,26 +280,23 @@ export class TableRenderer {
...
-          <i class="fa fa-search-plus"></i>
-        </a>`;
+      columnHtml += `<a class="table-panel-filter-link" data-link-tooltip data-original-title="Filter out value" data-placement="bottom"
+           data-row="${rowIndex}" data-column="${columnIndex}" data-operator="!=">`;
+      columnHtml += `<i class="fa fa-search-minus"></i>`;
+      columnHtml += `</a>`;
+      columnHtml += `<a class="table-panel-filter-link" data-link-tooltip data-original-title="Filter for value" data-placement="bottom"
+           data-row="${rowIndex}" data-column="${columnIndex}" data-operator="=">`;
+      columnHtml += `<i class="fa fa-search-plus"></i>`;
+      columnHtml += `</a>`;
```
Comment (torkelo) [inline public/app/plugins/panel/table/renderer.ts]: Can't see what your changing and why? What does this have to do with multi-line text?

**Comment by michael-az** [comment]
Sorry for not being clear enough on the matter.
The following screenshots should illustrate the current problem and the fix

The problem wasn't the indenting but rather the unnecessary line breaks which werent part of the data shown in the td, but added through the line breaks for the icons (probably unintended). 

before:
before
before2

after:
after

**Comment by michael-az** [comment]
Are there any updates?

**Comment by michael-az** [comment]
Are there any updates?

**Review by torkelo — APPROVED** [review]
